### PR TITLE
Add balances message.

### PR DESCRIPTION
**Pull request by biocom**
Screenshot 2020-04-09 at 14 56 24
Screenshot 2020-04-09 at 14 56 12

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr862--dexreact.review.gnosisdev.com

File: src/components/DepositWidget/Form.tsx [hunk 1]
```diff
@@ -105,6 +105,18 @@ export const Form: React.FC<FormProps> = (props: FormProps) => {
       <div>
         <h4>{title}</h4>
         <WalletDrawerInnerWrapper>
+          <div className="message">
+            This action consists of 2 steps:
+            <ol>
+              <li>{title} here.</li>
```
Comment (W3stside) [inline src/components/DepositWidget/Form.tsx:111]: can you remove the periods from the `li` text please

Comment (W3stside) [inline src/components/DepositWidget/Form.tsx:111]:
```suggestion
              <li>{title} here</li>
```

File: src/components/DepositWidget/Form.tsx [hunk 2]
```diff
@@ -105,6 +105,18 @@ export const Form: React.FC<FormProps> = (props: FormProps) => {
...
+              <li>Once the transaction is mined, wait for the batch to finish.</li>
```
Comment (W3stside) [inline src/components/DepositWidget/Form.tsx:112]:
```suggestion
              <li>Once the transaction is mined, wait for the batch to finish</li>
```

**Pull request merged**